README.markdown

```diff
@@ -22,13 +22,14 @@ Web posts are send using the HTTParty gem. The message is sent as the request bo
 
     messenger http://example.com "Message"
 
+Slack
+---
 
-Campfire
```

Comment:
err...Just kidding, I actually did accidentally remove the Campfire example. Will revert.